packages/x-charts-pro/package.json

```diff
@@ -53,7 +53,7 @@
   "peerDependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
-    "@mui/material": "^5.15.14",
+    "@mui/material": "^5.15.14 || ^6.0.0-beta.0",
```

Comment:
We never tested it with the first beta, so it'd be fair to set it to `^6.0.0-beta.6` 😅
Additionally, it would be great not to block the upgrade to stable, so I propose setting it to `^6.0.0-beta.6 || ^6.0.0`.
I tested a similar range on `@mui/x-data-grid` and it seems to look good:

<img width="947" src="https://github.com/user-attachments/assets/18448f4b-12c8-4603-bf13-5b660499cc9a">

Reply:
Sure. 👌 
Or even better—leave only `6.0.0` as we will probably not release this before `@mui/material` stable. 🤔

Reply:
> we will probably not release this before @mui/material stable. 🤔

Even if we do, I'm not sure we should support betas 🤔
So we are coming full circle 😅 [`b11d987` (#14142)](https://github.com/mui/mui-x/pull/14142/commits/b11d9878c90cf7b20cfc064dd2a7542a379f3acb)

Reply:
Yes, but I changed your approach because it didn't work. 🙈 
We need to use fuller version specifiers.
I just tried installing codesandbox packages locally. 😉